onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -200,7 +200,6 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       // CSE. For example, if A and B nodes both do Add operation with a same value but different initializers, by
       // default, CSE will not merge them, because the different initializers are represented by different NodeArg.
       if (session_options.config_options.GetConfigOrDefault(kOrtSessionOptionsDisableDoubleQDQRemover, "0") == "0"){
-        transformers.emplace_back(std::make_unique<IdenticalChildrenConsolidation>());
```

Comment:
Do you want to remove it or still run it in L2? All partitioning is done by the time L2 runs so any fusion to quantized node (XNNPACK) or compilation (NNAPI) would have completed by then and it should be safe to consolidate.

Reply:
It won't be necessary. That are few improvement also need to be done to that Optimizer to make it useful, but I don't have time to revisit anytime soon.